### DISCUSSION TITLE
obsidian和zotero连用（Citation插件介绍）

### DISCUSSION THREAD

**Discussion by cyddgh** [obsidianzh/forum#28] [Plugin第三方插件]
https://mp.weixin.qq.com/s/Ee_u-E_fDjSJpSHIbBOiJg
obsidian可以和zotero连用了，进而可以使用bib引用。我们需要使用的第三方插件是citation（https://github.com/hans/obsidian-citation-plugin ）。本文成稿仓促,如有错误之处,请在下方留言。

# 安装插件
方法1：在ob-设置-第三方插件-社区插件（浏览）-查询citation
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214221118.png)
如果无法加载插件，可以参考521xueweihan（https://github.com/521xueweihan/GitHub520 ）的方法，修改host文件。

方法2：我将下载好的插件已经上传到蓝奏云（https://wws.lanzous.com/iQkHdjdaone
 ），大家可以下载后，参照之前的文章，手动安装插件。
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214221920.png)

# 设置zotero
- 打开zotero，此时zotero必须要安装有Better BibTeX插件并已经启动该插件，安装步骤可以参考参考文末的​帮助文档。​
- 点击文件，导出文献库
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214222110.png)
- 选择“Better CSL JSON”（该选项是安装Better BibTeX插件之后才有的），点击ok，导出一个json文件。
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214223533.png)
- 文件名可以自行命名，我个人建议以英文命名，本例中命名为“My bib.json”。将文件保存到自己所有的ob库文件夹下。
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214223816.png)

# 设置ob中的Citation插件
- 在ob中，点击设置，找到Citation。
- 在path中输入刚导出的json文件绝对路径，例如“D:\OneDrive\Obsidian\Obsidian-Work\My bib.json”
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214224103.png)
- 另外上图第三项（Literature note folder）还需要我们设置一个文件夹。我们就在ob库文件夹下新建一个名为“Reading notes”的文件夹。
- 设置完成后（可能需要重启ob），在ob文档中按Ctrl+Shift+O，就可以开始浏览文献。
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214225217.png)
- 按Ctrl+Shift+E，就可以插入文献引用。
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214225244.png)
- 文献具体存放位置就是在上一步设置的文件夹中（本文中是“Reading notes”文件夹）。
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214225308.png)

# bib有什么用
bib可以快速完成论文的引用，具体可以去看看zettlr（https://www.zettlr.com ）的说明。目前我还没尝试使用pandoc来完成格式化。

王树义老师在【roam面对面】第3期：Roam的未来—人机协作（https://www.bilibili.com/video/BV1gK411K7G2 ）25分钟左右开始也有相关演示。

# 帮助文件：如何在zotero中安装Better BibTeX
Better BibTeX
- 安装说明：https://retorque.re/zotero-better-bibtex/installation 
- 下载地址：https://github.com/retorquere/zotero-better-bibtex/releases/
- 备用下载（下载后请解压缩）：https://wws.lanzous.com/iKkKnjdasbg

打开zotero，点击工具-插件
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214222741.png)

在弹出的窗口的右上角找到齿轮标志-选择“Install Add-on From File”
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214222841.png)

选择刚刚下载的文件，例如“zotero-better-bibtex-5.2.95.xpi”，点击install now
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214223112.png)

点击Restart，重启zotero，即可完成安装。
![](https://gitee.com/cyddgi/picture-store/raw/master/img/20201214223142.png)

**Comment by JJger148** [comment]
When I export my Markdown Text to word, word does not recognise the citations. 
Is there a way to fix that?

**Reply by cyddgh** [reply]
Maybe you can try pandoc.
https://retorque.re/zotero-better-bibtex/exporting/pandoc/

**Comment by fred1357944** [comment]
分享我自己學術研究工作流的影片😊
https://youtu.be/FNfBbj2G3_E
